system/security/shared/caching.cpp

```diff
@@ -478,7 +479,7 @@ void CPermissionsCache::removeFromUserCache(ISecUser& sec_user)
 
 bool CPermissionsCache::addManagedFileScopes(IArrayOf<ISecResource>& scopes)
 {
-    WriteLockBlock writeLock(m_scopesRWLock);
+    //WriteLockBlock writeLock(m_scopesRWLock);
```

Comment:
worth changing this to a comment, stating that this method must be called while m_scopesRWLock lock held.

Reply:
Agreed. I left is commented (in this draft) so you could see that the lock must be taken prior. I will change to a comment for the "real" PR.